pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm[toml]>=6.2", "numpy"]
```

Comment:
we need numpy for build? if so we might consider changing this to `"oldest-supported-numpy"` depending on the need. the oldest-supported is for api compat if I remember correctly but it's been a while since I delved into it deeply.

Reply:
nope, I added that to test stuff out because aipy wasn't building properly, but I think it should work without it.